### DISCUSSION TITLE
Making `Search` component on landing page functional

### DISCUSSION THREAD

**Discussion by mmailaender** [skeletonlabs/skeleton#2203] [Ideas]
I've found myself now multiple times clicking on the search component and trying to search there:

![image](https://github.com/skeletonlabs/skeleton/assets/87228994/83c025fc-7531-49a2-8e92-cc6eb5cf6e13)

Does this happen to anyone other than me?
What do you think about making this `Search` component functional?

**Comment by Mahmoud-zino** [comment]
I would suggest changing the search labels, since making the search input functional will require more work with little to no benefits.

**Reply by Sarenor** [reply]
Could just make it open the actual CTRL+K dialog on click, too 🤔

**Reply by endigo9740** [reply]
None of the components there are functional, it's just a preview. I'd suggest we change the label and placeholder to make it more obvious it's an example.
